src/SFML/Window/Unix/GlxContext.cpp

```diff
@@ -153,10 +155,11 @@ m_ownsWindow(false)
     // Save the creation settings
     m_settings = settings;
 
-    // Make sure that extensions are initialized if this is not the shared context
-    // The shared context is the context used to initialize the extensions
-    if (shared && shared->m_display)
-        ensureExtensionsInit(shared->m_display, DefaultScreen(shared->m_display));
+    // Open the connection with the X server
+    m_display = OpenDisplay();
+
+    // Make sure that extensions are initialized
+    ensureExtensionsInit(m_display, DefaultScreen(m_display));
```

Comment:
Nothing bad... GLX extension loading only requires a valid X display connection to be available. Since we open one up right before the call it will always succeed as long as the open succeeds.